### PR TITLE
v8 - update dependencies

### PR DESCRIPTION
**Pull request by Shazwazza**
Mvc, WebApi, Owin, MiniProfiler, Newtonsoft.Json, Automapper, etc... and the adjustments necessary

### CODE REVIEW

File: src/Umbraco.Core/Configuration/GlobalSettings.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ public class GlobalSettings : IGlobalSettings
         private static string _reservedPaths;
         private static string _reservedUrls;
         //ensure the built on (non-changeable) reserved paths are there at all times
-        internal const string StaticReservedPaths = "~/app_plugins/,~/install/,"; //must end with a comma!
+        internal const string StaticReservedPaths = "~/app_plugins/,~/install/,~/mini-profiler-resources/"; //must end with a comma!
```
Comment (nul800sebastiaan) [inline src/Umbraco.Core/Configuration/GlobalSettings.cs]: `//must end with a comma!`

It currently doesn't, also.. why must it, seems weird?

Comment (Shazwazza) [reply src/Umbraco.Core/Configuration/GlobalSettings.cs]: some old crappy code, i dunno, i didn't actually change that from v7, in v7 the static reserved paths also need this

**Comment by Shazwazza** [comment]
All updates are done

image

Some code changes were required for a couple of the updates like Mvc, WebApi, Automapper, MiniProfiler but they are all pretty minor. 

All tests are passing: https://umbraco.visualstudio.com/Umbraco%20Cms/_build/results?buildId=17456 and it works based on some fairly simple website tests.

The nuspec's have been updated but more importantly they are updated to have the correct target .net framework. Without this there is nothing preventing anyone from trying to install this into a .net 4.5 app! Also note that this should be added in v7 too (I have an outstanding/unmerged PR for that)

**Comment by clausjensen** [comment]
My local VS is acting up a bit (maybe resharper test runner?). I've reviewed the code, tested that basic functionality works (creating stuff - rendering on a site .. all that). So merged and pushed - seems to be building fine on the build server as well.

**Pull request merged**